### DISCUSSION TITLE
MacOS Unity C# Help!

### DISCUSSION THREAD

**Discussion by membranobruno** [NvChad/NvChad#2265] [Q&A] [answered]
Hi! I fall in another trip, and this time with connecting nvchad with development in Unity.
1) Mono installed. The path is /Library/Frameworks/Mono.framework/Versions/Current
2) omnisharp installed via Mason

<img width="1063" alt="Screenshot 2023-08-04 at 18 20 47" src="https://github.com/NvChad/NvChad/assets/58270462/7eabe014-28ac-4cd0-8463-b91211d46c1f">

3) add the configuration to lspconfig.lua

<img width="677" alt="Screenshot 2023-08-04 at 18 23 19" src="https://github.com/NvChad/NvChad/assets/58270462/b79c257e-625f-473d-8e8a-3d557aaf7c69">

4) Restart, open Unity script and nothing )

No autocompletion, no formatting, nothing

Thanks for any help!

**Comment by siduck** [comment, marked as answer]
check LspInfo command's output

**Comment by membranobruno** [comment]
thanks for the tip, now I know how to debug LSP. There was a problem with   `cmd = { "omnisharp" } ` parameter.
The next quest is to find the solution for this

<img width="657" alt="Screenshot 2023-08-04 at 21 33 06" src="https://github.com/NvChad/NvChad/assets/58270462/d6a524ea-9912-4f3a-82d4-d1e80cf27546">

Looks like omnisharp has not idea about Unity classes.

**Reply by JustADude29** [reply]
Hey! Have you found a way to get it working?
